aiozk/transaction.py

```diff
@@ -68,6 +69,15 @@ async def commit(self):
 
         return result
 
+    async def __aenter__(self):
+        return self
+
+    async def __aexit__(self, ext_type, exception, tb):
+        result = await self.commit()
+        if not result:
+            raise TransactionFailed
+        return result
```

Comment:
``ext_type`` seems misspelled. And ``return`` statement is redundant.

What about adding ``if`` branch for the case of ``ext_type`` is not None? That means an exception occurs inside ``async with`` statement.

Reply:
@rhdxmr yeah, all makes sense. fixed both